src/main/java/seedu/address/model/person/Medication.java

```diff
@@ -9,11 +9,10 @@
  */
 public class Medication {
     public static final String MESSAGE_CONSTRAINTS =
-            "Medication can only contain alphabets and the following special characters -> .()/- "
-                    + "(e.g., - A.  (Panadol).\n"
-                    + "It can be an empty string at the point of initialisation, "
-                    + "as medication may not be decided yet.\n";
-    public static final String VALIDATION_REGEX = "^[A-Za-z0-9\\s.()/-]*|^$";
+            "Medication must can include alphanumeric characters, spaces, periods, commas, parentheses, hyphens, "
+                    + "and slashes.\n"
```

Comment:
Minor nit, maybe should specify it as forward slashes, I'll amend it in my changes later.